cylc/flow/data_store_mgr.py

```diff
@@ -2366,7 +2368,7 @@ def delta_log_record(self, record: 'LogRecord') -> None:
         if not hasattr(w_delta, 'log_records'):
             w_delta.log_records = []
         w_delta.log_records.append(
-            PbLogRecord(level=record.levelname, message=record.message)
+            PbLogRecord(level=record.levelname, message=cstrip(record.msg))
```

Comment:
Unrelated, however, it would appear that `LogRecord.message` is created at the point of message formatting, whereas `LogRecord.msg` is more primitive.